S7.Net/PLCHelpers.cs

```diff
@@ -316,10 +337,12 @@ private object ParseBytes(VarType varType, byte[] bytes, int varCount, byte bitA
             }
         }
 
-        public byte[] GetPackage(object value)
+        public static byte[] GetPackage(object value)
```

Comment:
Please move static helpers outside plc class.

Reply:
Will do. Also, this breaks the API (even just making it static does), let me know how you feel about this. Also, it would probably make sense to have this internal as well.

Reply:
I think this is used only internally. What we shouldn't break are the methods used externally, the ones in the documentation.
Also we should really mark this helpers methods for the protocol as internal.

Reply:
I agree, but nevertheless it breaks API. I'm not sure if you're already using semantic versioning, but that would seem like a good idea to me, meaning that changes like this will increment the major version number.